src/Middleware/ResponseCaching/src/Internal/SendFileFeatureWrapper.cs

```diff
@@ -19,10 +19,10 @@ public SendFileFeatureWrapper(IHttpSendFileFeature originalSendFileFeature, Resp
         }
 
         // Flush and disable the buffer if anyone tries to call the SendFile feature.
-        public Task SendFileAsync(string path, long offset, long? length, CancellationToken cancellation)
+        public Task SendFileAsync(string path, long offset, long? count, CancellationToken cancellation)
```

Comment:
Revert this rename.

Reply:
It's defined as `count` on the interface, and the class itself is internal. What's the reason for not renaming this?

Reply:
This is good.